packages/webapp/src/views/footer.html

```diff
@@ -8,7 +8,7 @@
           href: "/cookies",
           text: "Cookies"
         },{
-          href: "https://www.gov.uk/guidance/biodiversity-gain-sites-register-terms-and-conditions",
+          href: termsAndConditions or "https://www.gov.uk/guidance/biodiversity-gain-sites-register-terms-and-conditions",
```

Comment:
does this look right? or a mistake?

Reply:
This is using the termsAndConditions variable - if it's not set then it will default to the string. I've set that variable on all the credits views to use the credits terms and conditions. All other pages are falling back to the default string. So I can't see a mistake.